errorstyle.css

```diff
@@ -17,9 +16,13 @@ h1 {
     margin: auto;
     padding: 10px;
     width: 300px;
-    font-family: 'Nunito', sans-serif;
     font-size: 22px;
 }
+a {
+    text-decoration: none;
+    color: #EEE;
+}
+
 #container {
```

Comment:
Rather than using `float:left` and `float: right` to align the content side-by-side, i would suggest looking at the css classes that other pages like the homepage or sign up page put content side-by-side and use those same css classes. they are part of the site-wide CSS that your page has access to and you shouldn't need to write your own CSS to do this.